features/contract-state/Totals.tsx

```diff
@@ -87,10 +89,29 @@ const Totals = () => {
     prettyTotalTokens: string = defaultMissingDataDisplay,
     prettyCollSymbol: string = "",
     prettyTokenSymbol: string = "",
-    getExchangeLinkCollateral: string = "https://app.uniswap.org/#/swap",
-    getExchangeLinkToken: string = "https://app.uniswap.org/#/swap",
+    getExchangeLinkCollateral: string = "",
+    getExchangeLinkToken: string = "",
     exchangeName: string = "Uniswap"
   ) {
+    const addTokenToMetamask = async () => {
+      // Add token to users metamask wallet.
+      if (provider == null) return;
+
+      // @ts-ignore
```

Comment:
Should this be an async call? I mention because you declare `addTokenToMetamask` as an async func

Reply:
it turns out it does not need to be async. I have removed it accordingly